### PR TITLE
Fix Deceased Patient bug in Clinical module, improve font consistency

### PR DESCRIPTION
**Pull request by larslemos**

### CODE REVIEW

**Comment by larslemos** [comment]
[Fix commonController](https://trello.com/c/rIev6qke/299-fix-commoncontroller)

File: app/clinic/controllers/patientCommonController.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+(function () {
```
Comment (ynurmahomed) [inline app/clinic/controllers/patientCommonController.js]: Duplicating the controller fixes the issue with ```patientCommonController``` not being loaded, but now we have to maintain both this one [and the one in registration](https://github.com/esaude/esaude-emr-poc/blob/master/app/registration/controllers/patientCommonController.js). The amount of work for this functionality is doubled. 

I think the delete button can be easily transformed into a directive, I'm available if you want to work together.

Comment (larslemos) [reply app/clinic/controllers/patientCommonController.js]: @ynurmahomed  Lets do it

File: test/spec/clinic/scheduleListController.spec.js [hunk 2]
```diff
@@ -1,29 +1,78 @@
...
+
+
+              var uuid = 'd00c4b2e-7c13-4d05-9f87-8a9c4b07ad52?providerUuid=8d4a4488-c2cc-11de-8d13-0010c6dffd0f';
+              var currentProvider = {
+                uuid: '8d4a4488-c2cc-11de-8d13-0010c6dffd0f'
+
+              }
+              $rootScope.currentProvider = currentProvider;
+              $httpBackend.whenGET("/poc_config/openmrs/i18n/common/locale_en.json").respond({});
+              $httpBackend.whenGET("/openmrs/ws/rest/v1/reportingrest/cohort/"+uuid).respond({});
```
Comment (ynurmahomed) [inline test/spec/clinic/scheduleListController.spec.js]: I think this expectation belongs in the service layer.

File: test/spec/clinic/scheduleListController.spec.js [hunk 3]
```diff
@@ -1,29 +1,78 @@
...
+
+
+
+              var uuid = 'd00c4b2e-7c13-4d05-9f87-8a9c4b07ad52?providerUuid=8d4a4488-c2cc-11de-8d13-0010c6dffd0f';
+              var currentProvider = {
+                uuid: '8d4a4488-c2cc-11de-8d13-0010c6dffd0f'
+
+              }
+              $rootScope.currentProvider = currentProvider;
+              $httpBackend.whenGET("/poc_config/openmrs/i18n/common/locale_en.json").respond({});
```
Comment (ynurmahomed) [inline test/spec/clinic/scheduleListController.spec.js]: We already know how to solve this. See [docs](https://angular-translate.github.io/docs/#/guide/22_unit-testing-with-angular-translate#unit-testing-with-angular-translate_the-solution) and [patientCommonController.spec.js](https://github.com/esaude/esaude-emr-poc/blob/master/test/spec/registration/patientCommonController.spec.js#L2).

File: test/spec/clinic/scheduleListController.spec.js [hunk 4]
```diff
@@ -1,29 +1,78 @@
...
+              $httpBackend.whenGET("/openmrs/ws/rest/v1/reportingrest/cohort/"+uuid).respond({});
+
+              // $rootScope.$apply();
```
Comment (ynurmahomed) [inline test/spec/clinic/scheduleListController.spec.js]: I think its better not to commit commented code.

**Pull request merged**